Source/monster.h

```diff
@@ -168,100 +167,115 @@ struct CMonster {
 extern CMonster LevelMonsterTypes[MaxLvlMTypes];
 
 struct Monster { // note: missing field _mAFNum
-	const char *mName;
-	std::unique_ptr<uint8_t[]> uniqueTRN;
-	AnimationInfo AnimInfo;
-	int _mgoalvar1;
-	int _mgoalvar2;
-	int _mgoalvar3;
-	int _mVar1;
-	int _mVar2;
-	int _mVar3;
-	int _mmaxhp;
-	int _mhitpoints;
-	uint32_t _mFlags;
+	const char *name;
+	std::unique_ptr<uint8_t[]> uniqueMonsterTRN;

```

Comment:
Why repeat `Monster` here? This member already belongs to the `Monster` struct.

Reply:
I'm not sure if the name `uniqueTRN` is enough to convey that this is the TRN for when the monster is a unique monster as opposed to the monster's very own TRN.

Reply:
I agree with @StephenCWills  - this `monster` in `uniqueMonsterTRN` is attached to the specific monster being unique and not to every monster in general. Removing the monster would create more confusing name, implying the TRN is unique, not the monster.